python/test/test_polars.py

```diff
@@ -129,6 +123,7 @@ def test_private_lazyframe_explicit_sum(measure):
 def test_private_lazyframe_sum(measure):
     pl = pytest.importorskip("polars")
     pl_testing = pytest.importorskip("polars.testing")
+    dp.enable_features("rust-stack-trace")
```

Comment:
If it's useful, I'd prefer to just add this to `test/__init__.py` since it will end up being applied globally.